pkg/3scale/amp/component/system_mysql.go

```diff
@@ -188,8 +188,8 @@ func (mysql *SystemMysql) Deployment(containerImage string) *k8sappsv1.Deploymen
 								},
 							},
 							Env: []v1.EnvVar{
-								helper.EnvVarFromSecret("MYSQL_USER", SystemSecretSystemDatabaseSecretName, SystemSecretSystemDatabaseUserFieldName),
-								helper.EnvVarFromSecret("MYSQL_PASSWORD", SystemSecretSystemDatabaseSecretName, SystemSecretSystemDatabasePasswordFieldName),
+								helper.EnvVarFromValue("MYSQL_USER", mysql.Options.User),
+								helper.EnvVarFromValue("MYSQL_PASSWORD", mysql.Options.Password),
 								// TODO This should be gathered from secrets but we cannot set them because the URL field of the system-database secret
 								// is already formed from this contents and we would have duplicate information. Once OpenShift templates
 								// are deprecated we should be able to change this.
```

Comment:
Should we remove this comment since we are now solely parsing these vars from values instead of secrets? Or should these be gathered from secrets?

CC @MStokluska